app/app.go

```diff
@@ -89,26 +90,41 @@ func Run(app App) int {
 		return 1
 	}
 
-	// register signals to kill the application
-	signals := make(chan os.Signal, 1)
-	signal.Notify(signals, syscall.SIGINT)
-	signal.Notify(signals, syscall.SIGTERM)
+	// register terminationSignals to kill the application
+	terminationSignals := make(chan os.Signal, 1)
+	signal.Notify(terminationSignals, syscall.SIGINT)
+	signal.Notify(terminationSignals, syscall.SIGTERM)
+
+	stackTraceSignal := make(chan os.Signal, 1)
+	signal.Notify(stackTraceSignal, syscall.SIGABRT)
```

Comment:
Currently we capture only SIGTERM, SIGINT and we don't do a [graceful shutdown](https://github.com/ava-labs/avalanchego/blob/c8a5d0b11bcfe8b8a74983a9b0ef04fc68e78cf3/vms/rpcchainvm/vm.go#L44) if we get SIGABRT. 

So, I thought that since SIGABRT is a useful way to print the stack trace, we might as well not shutdown violently while doing it, so I chose SIGABRT. 

I can change this to SIGUSR1 or something else, I also don't feel too strong about the choice of signal.